go-controller/pkg/node/gateway_udn.go

```diff
@@ -975,3 +984,56 @@ func (udng *UserDefinedNetworkGateway) removeDefaultRouteFromVRF() error {
 	}
 	return nil
 }
+
+// updateAdvertisedUDNIsolationRules adds nft rules to drop locally generated traffic destined to the UDN subnets for
+// BGP advertised networks. It blocks access to the full UDN subnet to handle a case in L3 when a node tries to access
+// a host subnet available on a different node. When isNetworkAdvertised is false the rules are removed.
```

Comment:
```suggestion
// a host subnet available on a different node. When isNetworkAdvertised is false the rules are removed.
```
this is in case we stop advertising the UDN on day2?